loggly/loggly.go

```diff
@@ -27,6 +28,7 @@ func init() {
 
 	r := &router.Route{
 		Adapter: "loggly",
+		FilterName: os.Getenv(filterNameEnvVar),
```

Comment:
I'm unfamiliar with this option (though I don't spend a ton of time with logspout), but what is the behavior when the environment variable is empty ("")?